src/quo2/components/information_box.cljs

```diff
@@ -0,0 +1,94 @@
+(ns quo2.components.information-box
+  (:require [quo.theme :as theme]
+            [quo.react-native :as rn]
+            [clojure.string :as string]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.button :as quo2.button]
+            [status-im.utils.handlers :refer [>evt <sub]]
+            [quo2.components.info-message :as info-message]))
+
+(def themes
+  {:light {:default     {:bg     colors/white
+                         :border colors/neutral-20
+                         :icon   colors/neutral-50
+                         :text   colors/black}
+           :informative {:bg     colors/primary-50-opa-5
+                         :border colors/primary-50-opa-10
+                         :icon   colors/primary-50
+                         :text   colors/black}
+           :error       {:bg     colors/danger-50-opa-5
+                         :border colors/danger-50-opa-10
+                         :icon   colors/danger-50
+                         :text   colors/danger-50}
+           :close-button colors/black}
+   :dark  {:default     {:bg     colors/neutral-90
+                         :border colors/neutral-70
+                         :icon   colors/neutral-40
+                         :text   colors/white}
+           :informative {:bg     colors/primary-50-opa-5
+                         :border colors/primary-50-opa-10
+                         :icon   colors/white
+                         :text   colors/white}
+           :error       {:bg     colors/danger-50-opa-5
+                         :border colors/danger-50-opa-10
+                         :icon   colors/danger-50
+                         :text   colors/danger-50}
+           :close-button colors/white}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(defn get-color-by-type [type key]
+  (get-in themes [(theme/get-theme) type key]))
+
+(defn close-information-box [id global? on-close]
+  (>evt [:close-information-box id global?])
+  (when on-close (on-close)))
+
+(defn information-box
+  "[information-box opts \"message\"]
+   opts
+   {:type            :default/:informative/:error
+    :closable?       true/false          ;; allow information box to be closed?
+    :id              :information-box-id ;; unique id (required for closable? information box)
+    :global?         true/ false         ;; close information box across all profiles
+    :icon            :main-icons/info    ;; information box icon
+    :style           style
+    :button-label    \"PressMe\"         ;; add action button with label
+    :on-button-press action              ;; (required for information box with button-label)
+    :on-close        on-close            ;; (optional on-close call)"
+  [{:keys [type closable? id global? icon style button-label on-button-press on-close]} message]
+  (let [closed?          (if closable? (<sub [:information-box-state id]) false)
```

Comment:
it would be better to abstract the component completely, so we need to pass closed as a parameter from outside the component